cursorless-talon/src/compound_targets.py

```diff
@@ -44,19 +44,19 @@ def cursorless_range(m) -> str:
         return primitive_targets[0]
 
     if len(primitive_targets) == 1:
-        start = BASE_TARGET.copy()
+        anchor = BASE_TARGET.copy()
```

Comment:
same with base target; we should prob just have a function that constructs one

Reply:
Yes do that

Reply:
https://github.com/cursorless-dev/cursorless/issues/719